src/BaseElement.js

```diff
@@ -238,6 +245,10 @@ class BaseElement extends HTMLElement {
 				if (JSON.stringify(oldValue) !== newValueString) {
 					this._state[property] = newValue;
 
+					if (newValue instanceof Store) {
+						newValue.subscribe(this);
+					}
+
```

Comment:
@quarkus this should be already merged from another branch ?!

Reply:
hmm . but its not !?

Reply:
Ok .. i looked into it again.

Usually an element is registered as a Store observer once its properties are defined.
The  above covers the case when a nested element requests a value which is an instance of Store.

To maintain reactivity we need to register that the store is injected to as an observer on some point.

I think we were discussing where to so it in the other Branches and ended up doing it here. 
The alternative was to subscribe the element  to the store within the "onRequestContext" method.

Reply:
... just lost another long comment. thanks github.

tldr;
Doing this in the setter has the advantage that it works without the DI Feature and would also work for runtime assignments of store instances to other components.

That is a good thing